xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/CategoriesCode.xml

```diff
@@ -315,12 +315,12 @@
   #set($nameUrl = $escapetool.url($name))
   #foreach($i in [1..$level])*#end ##
 &lt;span class="blog-category-level"&gt;&lt;span class="blog-category"&gt;##
-&lt;a href="$services.rendering.escape($xwiki.getURL('Blog.CategoryRss', 'view', "xpage=plain&amp;amp;category=$nameUrl"), $doc.syntax)" title="RSS"&gt;&lt;img class="icon icon-manage" src="$xwiki.getSkinFile('icons/xwiki/rss-medium.png')" alt="[RSS]"/&gt;&lt;/a&gt;##
-&lt;span class="wikilink"&gt;&lt;a href="$services.rendering.escape($xwiki.getURL($name), $doc.syntax)"&gt;#getCategoryName($xwiki.getDocument($name)) &lt;span class="itemCount"&gt;($totalEntries)&lt;/span&gt;&lt;/a&gt;&lt;/span&gt;&lt;/span&gt; ##
+&lt;a href="$services.rendering.escape($xwiki.getURL('Blog.CategoryRss', 'view', "xpage=plain&amp;amp;category=$nameUrl"), $doc.syntax)" title="RSS"&gt;#toolImage('feed','[RSS]')&lt;/a&gt;&amp;nbsp;##
```

Comment:
If we remove the `$alt` parameter of the macro, then you should change this call here.